### DISCUSSION TITLE
Changes to the Elexis websites (elexis.info)

### DISCUSSION THREAD

**Discussion by debenol1** [elexis/elexis.github.io#5] [Announcements]
After the previous website (elexis.info) was moved to Github (github.com/elexis/elexis.github.io) we now want to revise the presented content.

**We want to achieve the following goals:**

- Improving the user experience (where can I find which content?)
    - Improving access to content
    - Reducing the complexity of the content presented
    - Consistent presentation of content for different target groups such as users / sysOps / developers (manager are directed to Medelexis)

- Clear presentation of the different Elexis domains (elexis.info / elexis.ch / medelexis.ch)
    - Reflection of the the different domains
    - Presentation of the mutual respectful attitude between the different Elexis domains
   
- Comprehensible presentation of the various products/projects
    - Elexis Client
    - Elexis Environment
    - et al.

- Easier / faster maintenance of the presented content

**The following suggestions are made for achieving the objectives wihtin elexis.info:**
- Easy maintanable, single-page website (markdown)
- Only in english
- Focus on the following content
    - Presentation of the community
    - Presentation of the available documentations (also outside the elexis.info domain)
    - Presentation of the different products/projects
    - Presentation of the different resources

**The following suggestions are made for achieving the objectives wihtin github.com/elexis/elexis.github.io/wiki:**
- Redirect wiki.elexis.info to github.com/elexis/elexis.github.io/wiki
- Only in english (except handbooks)
- Focus on the following content
    - Information about the building process
    - Information (procedure/rules/conditions) about PRs
    - Information about the installation of Elexis
    - Release Notes 
    - Component grid: Information about the features/plugins whereas each component describes itself using Readme.md

**Comment by ngiger** [comment]
Hi

Thank you very much for your effort!

Am Donnerstag, 29. August 2024, 09:56:53 CEST schrieb Olivier Debenath:
> Easy maintanable, single-page website (markdown)
> Only in english
I do not object that technical/developer information is only presented in 
english, but I think elexis.info should be presented in the LOCAL language for 
newcomers which might not be fluent in english.
If you continue to maintain it as markdown , I think you could ask that 
maintaining other languages than german and enlish is only done if you find a 
maintainer for it (I would volunteer for french).

Best regards

Niklaus

**Reply by debenol1** [reply]
Hi

as you suggest, a significant aspect that makes the editorial maintenance of the current site complex is its multilingualism. In addition, Github does not support multilingualism by default; perhaps there is a reason for this? Multilingualism can only be implemented with great effort, as the functionality of the Jekyll multilingual plugin has to be recreated manually. An adventure that has already cost me quite a lot of time and that I do not want to pursue any further.

I also wonder which target group is actually interested in the Elexis open source domain? In any case, it will be IT-savvy people (possibly also doctors who have IT-savvy support) who certainly understand English. Otherwise, they would hardly move meaningfully in this milieu. And how large is the target group of people who neither know anything about Elexis nor understand English? I have the distinct feeling that it is not that large at the moment. And finally, the question: Isn't the need for information about a software product in one's own language one that is usually satisfied by providers like Medelexis?

I would really like it if these questions were discussed and answered in more detail!

Regards

Olivier
